src/pages/UserList.tsx

```diff
@@ -24,7 +24,7 @@ import PrimaryText from '../components/common/PrimaryText';
 const UserList = () => {
   const [tabIndex, setTabIndex] = useState(0);
 
-  const [data, switchData] = useFilter([]);
+  const [filterData, switchData] = useFilter();
```

Comment:
```suggestion
  const [filteredData, switchData] = useFilter();
```

 배열로 반환했기 때문에 어느 값이든 네이밍해서 사용이 가능하지만 `filteredData`, `filteredItem` 혹은 `filteredUserData` `filteredUserItem` 등등 이런 식으로 `filtered`가 포함된 이름이 좋지 않을까 싶습니다!!